src/kepler_model/estimate/model_server_connector.py

```diff
@@ -8,22 +8,31 @@
 from kepler_model.util.config import is_model_server_enabled, get_model_server_req_endpoint, get_model_server_list_endpoint, download_path
 from kepler_model.util.loader import get_download_output_path
 from kepler_model.util.train_types import ModelOutputType
+from kepler_model.train.profiler.node_type_index import discover_spec_values
 
+machine_spec_mount_path = "/etc/machine_spec"
```

Comment:
can we keep all model-server related settings under `/etc/kepler/models/` ?

Reply:
Okay. Just to note that will also update in https://github.com/sustainable-computing-io/kepler/pull/1684 to the same path.